src/conda_smithy.rst

```diff
@@ -0,0 +1,55 @@
+conda-smithy rerender
+=====================
+
+conda-forge  uses `conda-smithy` to re-render the feedstock and update the feedstock configuration.
+This action updates the `README`, CI configuration, the build matrix,
+fix problems with the feedstock configuration, etc.
+
+The bot issues re-rendering PRs every now and then,
+but sometimes the users may need to issue them manually instead waiting for the bot.
+Here is how, and when, to re-render.
+
```

Comment:
> Do I have call rerender by myself, or is it run always automatically on each PR, so nothing had to be done manually.
The text means to mean, it is dona automatically. When I understood you right before, if had to call it manually.

I will clarify that but here is the gist of it.

Re-renderings are done automatically by the bot. However, the bot cannot cover all recipes immediately when a re-render is needed (and it should not because that would bring `conda-forge` CIs to a halt!).

So we can/should re-render manually when updating a recipe, or modifying one of those functional parts of the recipe like the build matrix or the `yum_requirements.txt` file.

Note that if you are not in a hurry you can leave it to the bot, but that is usually not an option for most people. (I, for one, do not re-render just for re-rendering sake, nor I merge the bot re-rendering PRs, when the current version of the package will remain unchanged. I do that to save CI time for more important actions like updating a package version.)